WebApiBackend/WebApiBackend/Dto/AddUserToFlatDto.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebApiBackend.Model;
+
+namespace WebApiBackend.Dto
+{
+    public class AddUserToFlatDto
+    {
+        public User AddedUser { get; set; }
```

Comment:
The DTO cannot store a User object otherwise it will create an infinite loop between flat and user objects when converting to JSON. It instead needs to store a UserDTO object.

Reply:
Thanks to Nathan for the nice explanation.